### DISCUSSION TITLE
Auditd shows over 73k execution of "uname" for one day on standalone node

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#13509] [2.4]
### Version

2.4.90

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

64

### Storage for /

200G

### Storage for /nsm

1T

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello!

One of our partners installed auditd on standalone node and noticed that for 24 hours (23.40 to be specific) command `uname -m` and `uname` was run about 36.7k times each, what means that uname runs almost every second.
Both of them returns static information, that can be set as variable.

It looks like that both called by `bootstrap-salt.sh`, but i'm not sure.

Yes, this is not a bug, but it's a little bit strange behaviour, maybe it's better to set a pair of variables with the necessary information and not to run binary `uname` every time? 

Looks like this affects not only `uname`, because, for example, there are more commands like it: `virt-what`(ran 36k times), `systemctl --version` (36k), `rpm --eval  %{_host_cpu}` (36k, returns same data as `uname -m`).
All of them can run only once at startup and save their output to variables, so it can increase script's speed.

Thanks.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
> It looks like that both called by bootstrap-salt.sh, but i'm not sure.

bootstrap-salt.sh is not written by us, it's written by the saltstack project:
https://github.com/saltstack/salt-bootstrap

Is bootstrap-salt.sh constantly running?
